db/update.sql

```diff
@@ -27,3 +28,15 @@ ALTER TABLE `User`
   ADD `organization_web` varchar(255) DEFAULT NULL;
 
 -- -----------------------------------------------------------------------------  
+-- Events information table
+CREATE TABLE IF NOT EXISTS `Events` (
+  `event_id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,
+  `description` text NOT NULL,
+  `organizer` varchar(255) NOT NULL,
+  `event_date` int(11) NOT NULL,
+  `event_time` int(11) NOT NULL,
```

Comment:
What are the two fields `event_date`, `event_time` is for? Is it starting date or ending date of event?

How do we store date, time when you choose to stored as integer type? MySQL has [DATETIME](http://dev.mysql.com/doc/refman/5.5/en/datetime.html) type for storing date, time data